kustomize/skyline/base/deployment-apiserver.yaml

```diff
@@ -293,7 +293,7 @@ spec:
                 name: skyline-apiserver-secrets
                 key: default-region
         - name: skyline-apiserver-db-migrate
-          image: "docker.io/99cloud/skyline:latest"
+          image: "docker.io/99cloud/skyline:zed"
```

Comment:
can we use the [2023.1](https://hub.docker.com/layers/99cloud/skyline/2023.1/images/sha256-5c81150d82f275fc7afd104758927703a7e7e61104af11c119ff633f613fe16c?context=explore) tag here?